900.version-fixes/x.yaml

```diff
@@ -38,6 +38,7 @@
 - { name: xdg-utils,                   verpat: "20[0-9]{6}",                               snapshot: true }
 - { name: xdot,                        ver: "20121124",                                    snapshot: true }
 - { name: xdotool,                     verpat: "20[0-9]{6}.*",                             incorrect: true } # it's 3.YYYYMMDD; XXX: problem
+- { name: xdotool,                     ver: "3.20200618.1",                                incorrect: true } # the tag they pull is from 2016..
```

Comment:
Please limit this with `ruleset: openmandriva` and add `ruleset: openmandriva, untrusted: true` to ignore possible future incorrect versions.

Reply:
see b218eec